docs/Docker.md

```diff
@@ -0,0 +1,53 @@
+# Docker
+
+You may use docker images of the ADBench project without having to install dependencies on your PC.
+
+## Building docker image
+
+1. Clone repository
+2. Add your AD tools if nessasary
+3. Run `docker build -f <dockerfile name> -t <docker image name> .` from root directory of ADBench project  
+   e.g. `docker build -f some.Dockerfile -t adb-docker .`
+
+## Running docker image
+
+1. Create folder for results
+2. Run docker image with proper arguments.  
+   At docker image startup, the `/ADBench/run-wrapper.sh` script is executed. The following are possible launch options:  
+   - Option `-r|--run` to run benchmark (execute [global runner script](./Architecture.md#Global-Runner))  
```

Comment:
I think, that reference to the file `GlobalRunner.md` would be better here

Reply:
fixed